UKHomeOffice/sgmr-service
# SPCR-174 Swap node-sass to sass

## Description
Remove `node-sass` and install `sass`

sass (Dart Sass) is now the preferred option as LibSass (used with node-sass) is deprecated

## To test
- remove package-lock and node-modules
- `npm i`
- `npm start`
- > app should compile

You `should` see warnings about soon to be deprecated syntax - these should all be on the govuk-frontend code not on our spcr code. This is a known situation https://frontend.design-system.service.gov.uk/installing-with-npm/#requirements 


## Developer Checklist

\* Required






Ensure you delete the branch once the Pull Request is merged.
